src/dataLoaders/components/collectorsWizard/verify/VerifyCollectorsStep.tsx

```diff
@@ -57,11 +56,15 @@ export class VerifyCollectorStep extends PureComponent<Props> {
             configID={telegrafConfigID}
           />
         </DapperScrollbars>
-        <OnboardingButtons
-          onClickBack={onDecrementCurrentStepIndex}
-          nextButtonText="Finish"
-          className="data-loading--button-container"
-        />
+
+        <Overlay.Footer className="data-loading--button-container">
+          <Button
+            color={ComponentColor.Primary}
+            text="Finish"
+            type={ButtonType.Submit}
+            testID="next"
+          />
+        </Overlay.Footer>
```

Comment:
Unfortunately, we cannot put an `Overlay.Footer` here. This component is child of a `Overlay.Body` - see https://github.com/influxdata/ui/blob/master/src/writeData/components/PluginCreateConfigurationWizard.tsx#L143-L152 and https://github.com/influxdata/ui/blob/master/src/writeData/components/PluginAddToExistingConfiguration/Wizard.tsx#L133-L141

Per @daviddkkim the footer should not be inside the body.

There's two choices here. Add the button to the Footer component in two places. Or, just use a `<Button>` here with possibly some CSS to position it correctly.

Reply:
This makes sense yes! I just fixed it, used a flexbox instead seems to be working the same!